services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -3269,9 +3269,6 @@ private void deleteOneStoreVersion(String clusterName, String storeName, int ver
         return;
       }
       String resourceName = Version.composeKafkaTopic(storeName, versionNumber);
-      LOGGER.info("Deleting helix resource: {} in cluster: {}", resourceName, clusterName);
```

Comment:
We should not remove this as kill pushjob could be called without calling delete store version.

Reply:
Part of the `killOfflinePush` will delete the helix resourced. So, this is not removed but moved